backend/.env.example

```diff
@@ -9,3 +9,7 @@ AUTH0_ISSUER_BASE_URL=https://your-auth0-domain/
 CLOUDINARY_CLOUD_NAME=your-cloud-name
 CLOUDINARY_API_KEY=your-api-key
 CLOUDINARY_API_SECRET=your-api-secret
+
+# Stripe
+FRONTEND_URL=http://localhost:5173
+STRIPE_API_KEY=your-stripe-api-key
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/46#issuecomment-2423618327>#d63ac7</a><span> - 10/19/2024, 06:40 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The STRIPE_API_KEY should not be included in version-controlled files, even as a placeholder, to prevent accidental exposure of sensitive information.
</div>
<div id="fix">
<b>Fix:</b> Remove the STRIPE_API_KEY from the .env.example file and document the need for this key in a separate, non-version-controlled document or secure key management system.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -12,4 +12,3 @@ CLOUDINARY_API_SECRET=your-api-secret
 
  # Stripe
  FRONTEND_URL=http://localhost:5173
 -STRIPE_API_KEY=your-stripe-api-key
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged